src/form/bf_mx_lin.jl

```diff
@@ -95,33 +128,41 @@ function constraint_mc_load_power_balance(pm::LPUBFDiagModel, nw::Int, i, bus_ar
     cstr_p = []
     cstr_q = []
 
-    cstr_p = JuMP.@constraint(pm.model,
-        sum(p[a] for a in bus_arcs)
-        + sum(psw[a_sw] for a_sw in bus_arcs_sw)
-        + sum(pt[a_trans] for a_trans in bus_arcs_trans)
-        .==
-        sum(pg[g] for g in bus_gens)
-        - sum(ps[s] for s in bus_storage)
-        - sum(pd[d] for d in bus_loads)
-        - sum(gs.*w for gs in values(bus_gs))
-    )
-
-    cstr_q = JuMP.@constraint(pm.model,
-        sum(q[a] for a in bus_arcs)
-        + sum(qsw[a_sw] for a_sw in bus_arcs_sw)
-        + sum(qt[a_trans] for a_trans in bus_arcs_trans)
-        .==
-        sum(qg[g] for g in bus_gens)
-        - sum(qs[s] for s in bus_storage)
-        - sum(qd[d] for d in bus_loads)
-        + sum(bs.*w for bs in values(bus_bs))
-    )
-
-    con(pm, nw, :lam_kcl_r)[i] = cstr_p
-    con(pm, nw, :lam_kcl_i)[i] = cstr_q
```

Comment:
these need to be added back, you will need to change `cstr_p` and `cstr_q` instead the loop over the conductors to another variable e.g. `cp` and `cq`, and add `push!(cstr_p, cp)`, `push!(cstr_q, cq)` to after their creation for this to work correctly

Reply:
During the last update you got almost everything, but indavertantly left out putting these back
```julia
con(pm, nw, :lam_kcl_r)[i] = cstr_p
con(pm, nw, :lam_kcl_i)[i] = cstr_q
```